en/docs/choreo-concepts/connections.md

```diff
@@ -0,0 +1,28 @@
+# Connections
+
+Services can exist in two main forms: standalone and integrated. By connecting services, organizations can access and benefit from the combined strength of various features and capabilities. Choreo allows you to connect services using Connections. 
+
+Connections in Choreo allows you to integrate Choreo components, or to integrate Choreo components to external services or resources. Connections provide a simple and uniform way to integrate with services and resources. A Connection will include the set of parameters that the application requires to create a connection at runtime. The visibility level of the Connection and the service for which it is created determine these parameters. You can easily refer to these parameters in the application runtime.
+ 
+You can add Connections in different visibility levels: Organization, Project, and Component. The visibility levels are described below:
+
+## Organization Connections
+
+Organization Connections are Connections created for services within a particular organization. These Connections **can be used by any component within the organization**.
+
+For example, create an Organization Connection to share a database across the organization for all the projects and components to reuse. Components can refer to Organization Connections using the connection id. 
```

Comment:
Shall we replace all occurrences of id with ID based on https://developers.google.com/style/word-list#id

Reply:
done